README.md

```diff
@@ -8,33 +8,22 @@ Project is bootstrapped with [ts-starter-template](https://github.com/samhwang/t
 
 ### Setting up the bot in Discord
 
-- [Create a bot in discord](https://interactions-py.github.io/interactions.py/Guides/02%20Creating%20Your%20Bot/).
+- [Create a bot in discord](https://discordjs.guide/preparations/setting-up-a-bot-application.html#creating-your-bot).
 - Add that bot to your server.
-  - When making the invite through OAuth2 URL Generator, make sure to enable 
+  - When making the invite through OAuth2 URL Generator, make sure to enable
     - `bot` and `applications.commands` options
-    - all `Privileged Gateway Intents` options
-  - Follow the [Invite your bot](https://interactions-py.github.io/interactions.py/Guides/02%20Creating%20Your%20Bot/) section for reference.
+    - `Privileged Gateway Intents => Message Content Intent` options
```

Comment:
This has to be changed for the least-privileged principle. We shouldn't give the bots more permission than it needs to.